base/associative.jl

```diff
@@ -64,6 +64,8 @@ in(k, v::KeyIterator) = get(v.dict, k, secret_table_token) !== secret_table_toke
     keys(a::Associative)
 
 Return an iterator over all keys in a collection.
+`keys(Dict)` and `values(Dict)` both iterate the `Dict`
```

Comment:
`Dict` should be `a` for consistency with the signature (`Dict` is a type, not an object). By the way, can you fix `collect(keys(d))` below to use `a` too?